modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -188,30 +188,36 @@ Please consider alternatives to using this service, such as independently verify
 
 .Procedure: Installing an Errata Service
 
-. On the {productname} Server, from the command prompt, as root, add the ``sle-module-development-tools`` module:
+. On the {productname} Server host, from the command prompt, as root, add the [systemitem]``sle-module-development-tools`` module:
 +
 ----
-SUSEConnect --product sle-module-development-tools/15.2/x86_64
+mgrctl exec -- SUSEConnect --product sle-module-development-tools/15.5/x86_64
```

Comment:
To run the script on the host, the host needs to have the Development tools module. It should be checked if that works on SLE Micro 5.5 thought

Reply:
Ok, I'll change the description accordingly but need someone who can verify it.